src/backend/distributed/citus--7.0-3--7.0-4.sql

```diff
@@ -0,0 +1,25 @@
+/* citus--7.0-3--7.0-4.sql */
+
+SET search_path = 'pg_catalog';
+
+CREATE FUNCTION assign_distributed_transaction_id(initiatorNodeIdentifier int4, transactionNumber int8, transactionStamp timestamptz)
+     RETURNS void
+     LANGUAGE C STRICT
+     AS 'MODULE_PATHNAME',$$assign_distributed_transaction_id$$;
+ COMMENT ON FUNCTION assign_distributed_transaction_id(initiatorNodeIdentifier int4, transactionNumber int8, transactionStamp timestamptz)
+     IS 'Only intended for internal use, users should not call this. The function sets the distributed transaction id';
+
+CREATE OR REPLACE FUNCTION get_current_transaction_id(OUT databaseId oid, OUT processId int, OUT initiatorNodeIdentifier int4, OUT transactionNumber int8, OUT transactionStamp timestamptz)
+     RETURNS RECORD
+     LANGUAGE C STRICT
+     AS 'MODULE_PATHNAME',$$get_current_transaction_id$$;
+ COMMENT ON FUNCTION get_current_transaction_id(OUT databaseId oid, OUT processId int, OUT initiatorNodeIdentifier int4, OUT transactionNumber int8, OUT transactionStamp timestamptz)
+     IS 'returns the current backend data including distributed transaction id';
+
+CREATE OR REPLACE FUNCTION get_all_active_transactions(OUT databaseId oid, OUT processId int, OUT initiatorNodeIdentifier int4, OUT transactionNumber int8, OUT transactionStamp timestamptz)
+	RETURNS SETOF RECORD
+	LANGUAGE C STRICT
+	AS 'MODULE_PATHNAME', $$get_all_active_transactions$$;
+ COMMENT ON FUNCTION get_all_active_transactions(OUT databaseId oid, OUT processId int, OUT initiatorNodeIdentifier int4, OUT transactionNumber int8, OUT transactionStamp timestamptz)
+     IS 'returns all the active backends data including distributed transaction ids';
```

Comment:
"all the active backends data" doesn't seem useful to the user. Maybe "returns distributed transaction ids of active distributed transactions"